NEWS.rst

```diff
@@ -2,6 +2,23 @@
 Release Notes
 =============
 
+pkgdev 0.2.0 (2022-04-10)
+-------------------------
+
+- pkgdev commit: Mangle copyright header from single year into year range when
+  when (thanks to Thomas Bracht Laumann Jespersen, #49)
```

Comment:
last word should be appropriate, not double when

Reply:
Good catch!